arch/arm/core/Kconfig

```diff
@@ -45,6 +45,19 @@ config ARM_STACK_PROTECTION
 menu "Architectue Floating Point Options"
 depends on CPU_HAS_FPU
 
+config ARM_SECURE_FIRMWARE
+	bool
+	default n
+	help
+	  This option indicates that we are building a Secure application for
+	  an ARMv8-M-based MCU, which supports the Security Extension.
```

Comment:
remove the comma, and also Armv8-M

Reply:
Comma is ok, as for Arm, we have agreed to keep the style compliant with the earlier ARM architectures.

Reply:
[Agreed in #4983]